.github/workflows/onPRReviews.yml

```diff
@@ -1,23 +1,119 @@
 name: On review update
 on:
   pull_request:
-    types: [review_requested, review_request_removed]
+    types: [ready_for_review, review_requested, review_request_removed]
   pull_request_review:
     types: [edited, dismissed, submitted]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
 jobs:
-  checkMergeReady:
+  get-pr-number:
+    name: Get Current PR Number
+    runs-on: ubuntu-latest
+    outputs: 
+      pr-number: ${{ steps.condval.outputs.value }}
+    steps:
+      - uses: 3drepo/action-cond@v1
+        id: condval
+        with:
+          cond: ${{ github.event_name == 'pull_request' }}
+          if_true: ${{ github.event.number }}
+          if_false: ${{ github.event.pull_request.number }}
+
+  get-issue-number:
+    name: Get Current Issue Number
+    needs: [ get-pr-number ]
+    runs-on: ubuntu-latest
+    outputs: 
+      issue-number: ${{ steps.commitMsgParser.outputs.issue-number }}
+      content-id: ${{ steps.commitMsgParser.outputs.content-id }}
+    steps:
+      - name: Extract Issue Number from PR
+        uses: 3drepo/extract-issue-number-from-PR@v1
```

Comment:
Since you use this in so many different context I think you should keep this simple and only do what it says on the tin - extract the issue number base on the branch name.

Should probably move the is merged or not part out, it's confusing why we need to pass in a `base` if all we're doing is extract an issue number from a branch name

Reply:
renamed